src/test/java/org/folio/inventory/domain/items/ItemTest.java

```diff
@@ -21,4 +24,14 @@ public void cannotCreateItemIfStatusIsNull() {
     new Item("id", "holding-id", null, "material-type-id",
       "permanent-loan-type-id", new JsonObject());
   }
+
+  @Test
+  public void canMarkWithdrawnItemAsMissing() {
```

Comment:
Could it be API test instead? The scenario should be following:

`api.items.ItemApiExamples`:
1. Create an available item;
1. Mark it as withdrawn via calling API endpoint `/inventory/items/:itemId//mark-withdrawn` (see api.items.ItemMarkWithdrawnApiTest#markItemWithdrawn);
1. Mark item as missing by replacing the item (use `api.support.http.ResourceClient#replace(UUID, JsonObject)`);
1. Verify item status changed.

Reply:
Ok this should be done